commons/src/main/java/org/openmrs/eip/Utils.java

```diff
@@ -20,11 +21,14 @@ public class Utils {
 	/**
 	 * Gets a list of all watched table names
 	 * 
-	 * @return
+	 * @return a list of table names
 	 */
 	public static List<String> getWatchedTables() {
-		String watchedTables = AppContext.getBean(Environment.class).getProperty(Constants.PROP_WATCHED_TABLES);
-		return Arrays.asList(watchedTables.split(","));
+		Optional<String> watchedTables = Optional.ofNullable(AppContext.getBean(Environment.class).getProperty(Constants.PROP_WATCHED_TABLES));
+		if (watchedTables.isEmpty()) {
+			throw new EIPException("The property named " + Constants.PROP_WATCHED_TABLES + " is required");
```

Comment:
It might be better to have some better descriptive text for how to fix this error, e.g.,

```java
"The property " + Constants.PROP_WATCHED_TABLES + " must set to a comma-separated list of table names to watch"
```